app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -43,6 +56,13 @@ def validate_import_vm
     Gem::Version.new(api_version) >= Gem::Version.new('4')
   end
 
+  def submit_configure_imported_vm_networks(userid, vm_id)
```

Comment:
@gmcculloug actually this piece of code is "fixing" the state of the post-imported VM that it contains NICs with no network assigned. Logic here is best effort and tries to pick appropriate network to assign to all of them, to make the VM at least somewhat useful after the conversion process. Calling this logic upon a "normal" (properly configured, existing) VM could only be detrimental to its mode of operation. So I'm not really sure about moving it to the VM model..

@borod108 any opinions